pkg/asset/machines/nutanix/machines.go

```diff
@@ -203,6 +205,50 @@ func provider(clusterID string, platform *nutanix.Platform, mpool *nutanix.Machi
 		providerCfg.Categories = mpool.Categories
 	}
 
+	// DataDisks
```

Comment:
is this comment necessary or does it need to be elaborated upon?

Reply:
removed this comment line.